README.md

```diff
@@ -2,6 +2,61 @@
 
 ![CodeQL](https://github.com/intel-retail/automated-self-checkout/actions/workflows/codeql.yaml/badge.svg?branch=main) ![GolangTest](https://github.com/intel-retail/automated-self-checkout/actions/workflows/gotest.yaml/badge.svg?branch=main) ![DockerImageBuild](https://github.com/intel-retail/automated-self-checkout/actions/workflows/build.yaml/badge.svg?branch=main)  [![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/intel-retail/automated-self-checkout/badge)](https://api.securityscorecards.dev/projects/github.com/intel-retail/automated-self-checkout)
 
+
+## Table of Contents
+
+- [Prerequisites](#prerequisites)
+- [QuickStart](#quickstart)
+- [Documentation](#documentation)
+- [Known issues](#known-issues)
+- [Disclaimer](#disclaimer)
+- [Datasets & Models Disclaimer](#datasets--models-disclaimer)
+
+## Prerequisites
+
+- [Docker](https://docs.docker.com/engine/install/ubuntu/) 
+- [Manage Docker as a non-root user](https://docs.docker.com/engine/install/linux-postinstall/)
+- Intel hardware (CPU, GPU, dGPU)
+
+## QuickStart
+
+1. Build the project
+
+```
+make build-ovms-server
+make build-python-apps
+```
+
+2. Download sample videos
+```
+cd benchmark-scripts
+./download_sample_videos.sh
+```
+
+3. Run camera simulator:
+
+(Go back to the root directory)
+
+```
+cd ..
+./camera-simulator/camera-simulator.sh
```

Comment:
i think there is also a make target for running camera-simulator: `make run-camera-simulator`.